app-utils/bili-sync/spec

```diff
@@ -0,0 +1,3 @@
+VER=2.1.1
+SRCS="git::commit=tags/v$VER::https://github.com/amtoaer/bili-sync"
+CHKSUMS="SKIP"
```

Comment:
Please add a CHKUPDATE field so that we can track upstream updates - refer to the [aosc-findupdate documentation](https://github.com/AOSC-Dev/aosc-findupdate/tree/master/docs) for instructions.